src/service/resource.js

```diff
@@ -357,10 +365,18 @@ export class Resource {
     const viewport = this.resources_.getViewport();
     this.layoutBox_ = box;
 
+    const win = this.resources_.win;
+    const computedStyle = win./*OK*/getComputedStyle(this.element);
```

Comment:
Although it occurs to me that calling getComputedStyle would probably have to be done somewhere inside a this.vsync_.measure call I guess?